src/browser/components/MainPage.jsx

```diff
@@ -79,14 +79,25 @@ export default class MainPage extends React.Component {
   }
 
   getTabWebContents(index = this.state.key || 0, teams = this.props.teams) {
+    const allWebContents = remote.webContents.getAllWebContents();
+    const openDevTools = allWebContents.find((webContents) => webContents.getURL().includes('chrome-devtools') && webContents.isFocused());
```

Comment:
wouldn't [getfocusedwebcontents](https://electronjs.org/docs/api/web-contents#webcontentsgetfocusedwebcontents) be faster and easier?

Reply:
That would pick up the webview in some cases, but that would work for the other pages! Thanks for the tip!